pages/home/oracle-integrity-staking/slashing-rulebook.mdx

```diff
@@ -0,0 +1,106 @@
+- **Data Collection:** Publishers are responsible for computing price data. This could involve accessing data from exchanges, trading platforms, or other price information sources
+- **Data Submission:** Publishers submit their respective measurement of price to the Pyth Network, ensuring a constant stream of accurate and timely information
+- **Maintaining Data Quality:** The most crucial role of publishers is ensuring the quality and integrity of the data they provide. This involves:
+  - **Reliable Sources:** Sourcing data from the necessary sources to produce high quality measurements of price data
+  - **Data Validation:** Implementing validation processes to verify the accuracy of the data before submission
+  - **Transparency:** Providing transparency into their data sources and methodologies so that users of the Pyth Network can assess the reliability of the data
+  - **Timeliness:** Ensuring data is submitted in a timely manner to reflect real-time market conditions
+
+### Responsibility for Data Quality
+
+Data publishers have a significant responsibility for maintaining data quality within the Pyth Network. This responsibility is not just ethical, but also incentivized through the network's structure:
+
+- **Staking and Slashing:** Publishers can stake PYTH tokens as collateral. In the event of data inaccuracies or inconsistencies, a portion of this stake can be slashed as a penalty. This mechanism ensures that publishers are discouraged from providing inaccurate data
+- **Reputation:** The reputation of data publishers within the Pyth Network is directly linked to the quality of data they provide. Publishers with a track record of providing accurate data are more likely to rank higher and attract more adoption
+
+### Importance of Data Quality
+
+The Pyth Network is designed to provide reliable and trustworthy financial data to decentralized applications and smart contracts. The quality of the data it provides is paramount for the functioning of these applications and the overall success of the network. Therefore, the role of data publishers and their commitment to data quality cannot be overstated.
+
+## Staking
+
+Staking through OIS is a program to enhance the security of price feeds produced by publisher on the Pyth Network.
+
+Per the design of OIS, a pool aligned with a data publisher accepts stake from the moment such publisher declares its stake account through configuration. The additional stake allows such publisher to earn a delegation fee to contribute to the costs of providing data to the Pyth Network. Data publishers compete with one another to increase the number of price feeds they contribute to, and/or increase the quality of data that allows them to be selected to publish and/or enable higher staking cap for the pool each publisher is aligned to.
```

Comment:
inaccurate, any pool can receive stake even if the publisher is opted-out

Reply:
a pool aligned with a data publisher accepts stake from the publisher itself or from delegators